common/network/uri_parser.c

```diff
@@ -16,6 +16,10 @@ bool uri_parse(char const *const uri, char *const scheme,
   size_t length = 0;
   size_t offset = 0;
   char *delim = NULL;
+  enum URI_CASE {
+    ipv4,
+    ipv6
+  } uri_case;
```

Comment:
Could you put this in `common/network/network.h` ?
```
  typedef enum ip_version_e {
    IPV4,
    IPV6
  } ip_version_t;
```

Reply:
ok